src/components/TextInput/__tests__/TextInput.spec.js

```diff
@@ -139,7 +139,7 @@ describe('Text input', () => {
       const button = getByRole('button', { name: /copy/i });
       expect(button).toBeInTheDocument();
 
-      document.execCommand = jest.fn();
+      document.execCommand = vi.fn();
```

Comment:
The only noticeable change in the test API when switching from jest to vitest was just replacing all use of the `jest` object to instead use `vi`.